### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by neetukumari4858**
Added Authentication
- Sign-up Page
I can see a sign-up page from where I can sign-up using my email, first name, last name, password & confirm password.
- Login Page
 I can see a login page from where I can log in using my email & password.
- Logout
I can see a logout button from where I can logout from the app

### CODE REVIEW

File: src/Components/Navigation/Navbar.js [hunk 1]
```diff
@@ -1,23 +1,32 @@
...
-    
-        </div>
-    </nav> 
+            <div className="nav-section">
+                <div className="outer-search-bar-div">
+                    <input type="text" className="search-bar" placeholder="   Search.." />
+                </div>
+                {token && user ? <button className='login-btn text-color' onClick={logoutHandler}>Logout</button> : <button className='login-btn'><Link className="nav-link" to="loginPage">Login</Link></button>
+                }
+                {/* <button className='login-btn'><Link className="nav-link" to="loginPage">Login</Link></button> */}
```
Comment (NamrataRaikwar2002) [inline src/Components/Navigation/Navbar.js]: You can remove these comments.

File: src/Page/LoginPage/Login.js [hunk 2]
```diff
@@ -1,30 +1,175 @@
...
     </div>
   )
 }
-export {Login}
+export { Login }
+
+
+
+
+// import  React ,{ useState } from 'react';
```
Comment (NamrataRaikwar2002) [inline src/Page/LoginPage/Login.js]: Remove this commented code.

**Review by NamrataRaikwar2002 — APPROVED** [review]
- Code is clean and readable
 Great work
one suggestion given

File: src/Context/AuthContext.js [hunk 3]
```diff
@@ -6,10 +6,13 @@ const AuthProvider=({children})=>{
     const [isLogedIn,setLogedIn]=useState(userLoggedIn)
     const [userDetail,setUserdetail]=useState({token:"",user: {}})
     const token=localStorage.getItem("token")
+    console.log(token,"gettoken")
     const user=JSON.parse(localStorage.getItem("user"))
-
+    console.log(token ,user,"now")
```
Comment (Tanjum786) [inline src/Context/AuthContext.js]: remove console logs

File: src/Context/AuthContext.js [hunk 4]
```diff
@@ -6,10 +6,13 @@ const AuthProvider=({children})=>{
...
+ 
     useEffect(() => {
         if (token && user){
+            console.log(token,"tokenuseEffect")
```
Comment (Tanjum786) [inline src/Context/AuthContext.js]: here same as above

File: src/Page/LoginPage/Login.js [hunk 5]
```diff
@@ -1,30 +1,175 @@
...
+//             <button type='button' className="videologin-btn" onClick={() => loginBtnHandler()}>Login</button>
+//             <h4 className='create-account'><Link to="/SignUpPage" className='createAccount'>Create New Account</Link> <i className='fas fa-angle-right'></i></h4>
+//           </div>
+//         </form>
+
+//       </div>
+//     </div>
+//   )
+// }
+// export { Login }
```
Comment (Tanjum786) [inline src/Page/LoginPage/Login.js:174]: remove the commented code if it is not useable

**Review by Tanjum786 — APPROVED** [review]
suggested some changes otherwise all look good

File: src/Components/Navigation/Navbar.js (hunk 1 above)
Comment (neetukumari4858) [reply src/Components/Navigation/Navbar.js]: okey @NamrataRaikwar2002 I will remove

File: src/Context/AuthContext.js (hunk 3 above)
Comment (neetukumari4858) [reply src/Context/AuthContext.js]: > remove console logs

thx @Tanjum786  for review I will remove console

**Pull request merged**